lib/vue-simple-suggest.vue

```diff
@@ -4,28 +4,40 @@
     @keydown.tab="isTabbed = true"
   >
     <div class="input-wrapper" ref="inputSlot"
+      role="combobox"
+      aria-haspopup="listbox"
+      aria-owns="suggestions"
+      :aria-expanded="!!listShown && !removeList ? 'true' : 'false'"
       :class="styles.inputWrapper">
       <slot>
         <input class="default-input" v-bind="$attrs" :value="text || ''"
+          aria-autocomplete='list'
+          aria-controls="suggestions"
+          :aria-activedescendant="!!hovered ? getId(hovered, hoveredIndex) : ''"
           :class="styles.defaultInput">
       </slot>
     </div>
     <transition name="vue-simple-suggest">
-      <div class="suggestions" v-if="!!listShown && !removeList"
+      <ul id="suggestions" class="suggestions" v-if="!!listShown && !removeList"
```

Comment:
This `[id]` attribute can be a trouble if there are multiple vue-simple-suggests on the page (which is very often the case).

Please, consider replacing it with a computed like `:id="_uid + '-suggestions'"`, or removing it. 🙂

Reply:
Every vue instance has a "private" `_uid` property. It's incremented globally by vue for each new rendered component, so it is unique for every vue instance.

So, in a case of the following layout:
```html
// root-component.vue
<template>
  <child-one ref="one"/>
  <child-two ref="two"/>
</template>

<script>
export default {
  mounted() {
    console.log(this._uid); // => 1
    console.log(this.$refs.one._uid); // => 2
    console.log(this.$refs.two._uid); // => 3
  }
}
</script>
```

This allows to uniqely identify any vue instance at a constant time, while also retining information about the id.

It's clear that using `Math.random()` we could also save the value and bind it to the id. However, there's still a relatively high chance of Id collisions, since randomizer does not "remember" any old results it has already generated.

We could also implement a system similar to Vue's `_uid`, incementing a global variable on `created()` of each `vue-simple-suggest`. But why bother reinventing the wheel?

Reply:
So, you can just basically do `:id="_uid + '-suggestions'"`, and it will work.

Reply:
Thanks! I did not know this, will implement.

Reply:
Nice!